src/azure-cli/azure/cli/command_modules/acr/task.py

```diff
@@ -982,3 +989,11 @@ def _get_trigger_event_list_patch(cmd,
             if SourceTriggerEvent.pullrequest.value in source_trigger_events:
                 source_trigger_events.remove(SourceTriggerEvent.pullrequest.value)
     return source_trigger_events
+
+
+def _get_branch_name(context_path):
+    # Context Path formats https://docs.docker.com/engine/reference/commandline/build/#git-repositories
+    branch = re.search(r'(?<=.git#)([^:/\n\s]*)', context_path)
```

Comment:
DevOps doesn't have `.git` in the path.  You might just look for the first occurrence of `#`.